pkg/webhook/utils/utils.go

```diff
@@ -53,3 +58,20 @@ func ValidateDefinitionRevision(ctx context.Context, cli client.Client, def runt
 	}
 	return nil
 }
+
+// ValidateCueTemplate validate cueTemplate
+func ValidateCueTemplate(cueTemplate string) error {
```

Comment:
Unit test should be added to this function.

Reply:
ok, I added unit test and fix CI error.